src/main/java/tlylt/haha/TaskList.java

```diff
@@ -104,13 +104,6 @@ void listFromDB() {
         }
     }
 
-<<<<<<< HEAD
-    /**
-     * Marks selected task as done.
-     *
-     * @param inputNum String for parsing task number.
-     */
-=======
     private void findFromDB(String keyword, Ui ui) {
         System.out.println("Here are the matching tasks in your list:");
         boolean hasRelated = false;
```

Comment:
Hi Wang Tao, thank you for your review.
In this case, I am referring to whether "database" **has related** tasks instead of whether a task **is related** to the search string, hence I think "hasRelated" is perhaps more grammatically correct in this sense?
Nonetheless, I would like to thank you for your time and I will check if my other boolean values could be better named.